libs/barista-components/slider/src/slider.spec.ts

```diff
@@ -493,8 +493,7 @@ describe('DtSlider', () => {
       } = getElements(fixture);
       const { slider } = testComponent;
 
-      //<any, any> is needed because _updateSlider is private
-      const update = jest.spyOn<any, any>(slider, '_updateSlider');
+      const update = jest.spyOn(slider, '_updateSlider');
```

Comment:
@SkaillZ said: This lead to errors when trying to build with Bazel

Reply:
it works this way:
```suggestion
      const update = jest.spyOn(slider as any, '_updateSlider');
```
then you don't have to change the class.